bin/stencil-release.js

```diff
@@ -1,13 +1,11 @@
 #!/usr/bin/env node
 
 require('colors');
-const release = require('../lib/release/release');
+const StencilRelease = require('../lib/release/release');
 const { PACKAGE_INFO } = require('../constants');
 const program = require('../lib/commander');
-const { checkNodeVersion } = require('../lib/cliCommon');
+const { printCliResultErrorAndExit } = require('../lib/cliCommon');
 
 program.version(PACKAGE_INFO.version).parse(process.argv);
 
-checkNodeVersion();
```

Comment:
The main reason to have the `checkNodeVersion` outside the main code is so it is not limited by the node version required by the repo.

eg. If as part of `StencilRelease` class we have `??` which is not supported in node 10, the node check won't run either in node 10.

This makes the `nodeCheck` pretty useless, because it might only run in the version it supports 😅

Reply:
Sounds reasonable to me. We have another place, that has same logic https://github.com/bigcommerce/stencil-cli/blob/master/lib/stencil-start.js#L98

Reply:
I'd definitely recommend moving that to the top of the file to avoid the issue I mentioned before.